zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/InterpreterGroup.java

```diff
@@ -168,13 +172,7 @@ public void close(String sessionId) {
     List<Interpreter> intpForSession = this.get(sessionId);
     close(intpForSession);
 
-    if (remoteInterpreterProcess != null) {
-      remoteInterpreterProcess.dereference();
-      if (remoteInterpreterProcess.referenceCount() <= 0) {
-        remoteInterpreterProcess = null;
-        allInterpreterGroups.remove(id);
-      }
-    }
+    makeSureRemoteProcessTerminate();
```

Comment:
I think `void close(String sessionId)` supposed not necessarily terminate process while an interpreter process can have multiple sessions.